### ISSUE TITLE
Convert Formula to English Text?

### ISSUE THREAD

**Issue by BSalita** [omarsar/deep_learning_notations#1]
I see that you have an English text explanation of notations. Are you aware of any project or web site that reads a formula and automatically creates a description? That would be a good teaching and validation tool.

This is a great notebook. I'll promote it in our Paris Machine Learning Study group.

**Comment by omarsar** [comment]
@BSalita You are right! And that is a great idea. Perhaps a good initiative would be to build a dictionary of some sort. I would keep that in mind as I enhance and add to this project. 

BTW, I haven't really seen any such material or resource online. 
Thanks for sharing! I appreciate it! 💯
